src/Files.App/Program.cs

```diff
@@ -273,20 +259,10 @@ public static void OpenShellCommandInExplorer(string shellCommand, int pid)
 
 		public static void OpenFileFromTile(string filePath)
 		{
-			IntPtr eventHandle = CreateEvent(IntPtr.Zero, true, false, null);
-
-			Task.Run(() =>
+			STATask.RunAsSync(() =>
```

Comment:
Is there a reason why async void isn't desirable here?

Reply:
So you mean adding Wait() in that helper method instead?

Reply:
Id like to keep this PR's scope just wrapping the unsafe code into a separate method.